threads/src/main/java/org/jboss/as/threads/ThreadSubsystemResourceDefinition.java

```diff
@@ -5,53 +5,38 @@
 
 package org.jboss.as.threads;
 
-import org.jboss.as.controller.AttributeDefinition;
-import org.jboss.as.controller.PersistentResourceDefinition;
 import org.jboss.as.controller.ReloadRequiredRemoveStepHandler;
+import org.jboss.as.controller.SimpleResourceDefinition;
 import org.jboss.as.controller.descriptions.DeprecatedResourceDescriptionResolver;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
 import org.jboss.as.controller.registry.OperationEntry;
 
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-
 /**
  * {@link org.jboss.as.controller.ResourceDefinition} for the root resource of the threads subsystem.
  *
  * @author Brian Stansberry (c) 2011 Red Hat Inc.
  */
 @Deprecated(forRemoval = true)
 @SuppressWarnings({"removal", "DeprecatedIsStillUsed"})
-class ThreadSubsystemResourceDefinition extends PersistentResourceDefinition {
+class ThreadSubsystemResourceDefinition extends SimpleResourceDefinition {
```

Comment:
@pferraro 

I would like to understand the usage of PersistentResourceDefinition better.
Could you explain when a resource does not need to be managed as a PersistentResourceDefinition? I don't get the reason for this change yet, which in turn cascades it to the other resources

Reply:
See https://issues.redhat.com/browse/WFCORE-6864